### DISCUSSION TITLE
Может кто поделиться, или поправить аргумент под инстаграм и фейсбук?

### DISCUSSION THREAD

**Discussion by Protey** [hufrea/byedpi#178] [General]
Нашел этот аргумент "s1 -o1 -Ar -o1 -At -f-1 -r1+s -As", вроде как твиттер и остальные мне нужные работают, но инстаграм и фейсбук не хочет работать.
Может кто поделиться, или поправить аргумент под инстаграм и фейсбук?

**Comment by barmaley6699** [comment]
s1 -o1 -Ar -o3 -At -f-1 -r1+s -As - у меня так работает
